lib/operations/collection/get.js

```diff
@@ -26,28 +26,28 @@ function validateProperties(map, properties) {
 	let status = 200;
 
 	const hasPageSize = properties.hasOwnProperty('pageSize');
-	const pageSizeIsNumber = Number.isInteger(properties.pageSize) && typeof (properties.pageSize) === 'number';
+	const pageSizeIsNumber = Number.isInteger(properties.pageSize);
 	const pageSizeIsValidValue = properties.pageSize > 0 && properties.pageSize === parseInt(properties.pageSize, 10);
 	const hasPageAfter = properties.hasOwnProperty('pageAfter');
 	const pageAfterKeyFound = map[properties.pageAfter];
 	const hasPageNum = properties.hasOwnProperty('pageNum');
-	const pageNumIsNumber = Number.isInteger(properties.pageNum) && typeof (properties.pageNum) === 'number';
+	const pageNumIsNumber = Number.isInteger(properties.pageNum);
 	const pageNumIsValidValue = properties.pageNum > 0 && properties.pageNum === parseInt(properties.pageNum, 10);
 
 	if (hasPageSize) {
 		if (!pageSizeIsNumber) {
-			status = 415; // Unsupported Media Type: Not a number
+			status = 400; // Bad Request: Not a number
 		} else if (!pageSizeIsValidValue) {
-			status = 422; // Unprocessable Entity: Invalid number
+			status = 400; // Bad Request: Invalid number
 		} else if (hasPageAfter && !pageAfterKeyFound) {
-			status = 404; // Not found
+			status = 400; // Bad Request: PageAfter not found
```

Comment:
I think you're right, 404 is probably most correct in this case. 
The most likely scenario of it occurring might be if somebody tries to access a resource which has been removed which would be a 404.
I have changed it back to 404 now.